--- a/phpmailer/DSNConfigurator.php
+++ b/phpmailer/DSNConfigurator.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * Configure PHPMailer with DSN string.
--- a/phpmailer/Exception.php
+++ b/phpmailer/Exception.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * PHPMailer exception handler.
--- a/phpmailer/OAuth.php
+++ b/phpmailer/OAuth.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 use League\OAuth2\Client\Grant\RefreshToken;
 use League\OAuth2\Client\Provider\AbstractProvider;
--- a/phpmailer/OAuthTokenProvider.php
+++ b/phpmailer/OAuthTokenProvider.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * OAuthTokenProvider - OAuth2 token provider interface.
diff -u b/phpmailer/PHPMailer.php b/phpmailer/PHPMailer.php
--- b/phpmailer/PHPMailer.php
+++ b/phpmailer/PHPMailer.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * PHPMailer - PHP email creation and transport class.
@@ -4281,13 +4281,6 @@
         }
 
         if (array_key_exists($key, $this->language)) {
-            if ('smtp_connect_failed' === $key) {
-                //Include a link to troubleshooting docs on SMTP connection failure.
-                //This is by far the biggest cause of support questions
-                //but it's usually not PHPMailer's fault.
-                return $this->language[$key] . ' https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting';
-            }
-
             return $this->language[$key];
         }
 
--- a/phpmailer/POP3.php
+++ b/phpmailer/POP3.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * PHPMailer POP-Before-SMTP Authentication Class.
--- a/phpmailer/SMTP.php
+++ b/phpmailer/SMTP.php
@@ -19,7 +19,7 @@
  * FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-namespace PHPMailer\PHPMailer;
+namespace Advancedform\PHPMailer;
 
 /**
  * PHPMailer RFC821 SMTP email transport class.
